### PR TITLE
Restik apis 

### PR DESCRIPTION
**Pull request by saumanbiswas**

### CODE REVIEW

File: client/clientset/fake/restik.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
+package fake
```
Comment (sadlil) [inline client/clientset/fake/restik.go]: rename this file to backup.go

File: client/clientset/restik.go [hunk 2]
```diff
@@ -0,0 +1,114 @@
+package clientset
```
Comment (sadlil) [inline client/clientset/restik.go]: backup file name

**Review by sadlil — CHANGES_REQUESTED** [review]

File: client/clientset/restik.go (hunk 2 above)
Comment (saumanbiswas) [inline client/clientset/restik.go]: rename restik.go -> backup.go ?

Comment (sadlil) [inline client/clientset/restik.go]: y

**Review by sadlil — APPROVED** [review]
LGTM

**Pull request merged**